README.rst

```diff
@@ -191,49 +189,36 @@ meta taking priority.
             #     'download_slot': 'quotes.toscrape.com'
             # }
 
-There is a shortcut, in case a request uses the same parameters as
-defined in the ``ZYTE_API_DEFAULT_PARAMS`` setting, without any further
-customization - the ``zyte_api`` meta key can be set to ``True`` or ``{}``:
+``ZYTE_API_DEFAULT_PARAMS`` does not make requests automatically go through
+Zyte Data API. See :ref:`enabled`.
 
-.. code-block:: python
+Parameters in ``ZYTE_API_DEFAULT_PARAMS`` are merged with parameters set via
+the ``zyte_api`` meta key, with the values in meta taking priority.
 
-    import scrapy
 
+.. _enabled:
 
-    class SampleQuotesSpider(scrapy.Spider):
-        name = "sample_quotes"
+Controlling which requests go through Zyte Data API
+---------------------------------------------------
 
-        custom_settings = {
-            "ZYTE_API_DEFAULT_PARAMS": {
-                "browserHtml": True,
-            }
-        }
+The ``ZYTE_API_ENABLED`` setting can be used to control whether all, none, or
+some requests go through Zyte Data API. It supports the following values:
```

Comment:
I think it might be better to have a separate option for enabling all requests to go through Zyte API, because of the following reasons:

1. Usually ..._ENABLED options just turn on/off the middleware.
2. It may also be tricky to set this setting to None from the command line, if I recall correctly. 
3. We shoud consider future Zyte API features, with data extraction use cases: if user only wants the extraction features, they'd need to set ZYTE_API_ENABLED to None, which is a bit strange.
4. Instead of relying on the user to specify correct request parameters to use Zyte API as a downloader, it'd be nice to set them on our side. This may involve not only passing static default parameters, but also taking request method, body and headers in account.

Reply:
3 is not fully clear to me, but I don’t think it matters at this point.

Shall we deal with 4 on a follow-up pull request, or here? I would love to work on it, but I also would like to discuss the API a bit before working on an implementation.